src/extension/app/utils/admin-client.js

```diff
@@ -105,6 +105,24 @@ export class AdminClient {
     return this.RATE_LIMITER.NONE;
   }
 
+  /**
+   * Shows the error toast.
+   * @param {string} message The error message
+   * @param {string} variant The toast variant (positive, warning, negative)
+   */
+  showErrorToast(message, variant) {
+    this.appStore.showToast(
+      message,
+      variant,
+      undefined,
```

Comment:
Not something for this PR, but seems like a good reason to move to passing an object instead ;)

Reply:
I was thinking the same thing :)